### PR TITLE
Aws lb obj

### PR DESCRIPTION
**Pull request by abhijitherekar**
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: specs/+cloudloadbalancerroutedata.spec [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Attributes
+attributes:
+  v1:
+  - name: healthStatus
+    description: The status of the target.
+    type: string
+    exposed: true
+    stored: true
+    example_value: healthly
+
```
Comment (dstiliadis) [inline specs/+cloudloadbalancerroutedata.spec:20]: We don't need to track health status. If defined we assume at some point it becomes healthy.

File: specs/+cloudtargetgroup.spec [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Model
+model:
+  rest_name: cloudtargetgroup
+  resource_name: cloudtargetgroups
```
Comment (dstiliadis) [inline specs/+cloudtargetgroup.spec:4]: Why do we need the target group? I don't think we care about it in any way. May be I am missing something but given the load balancer and the route data we are good.

**Review by dstiliadis — APPROVED** [review]

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "aws-lb",
    "component": "gaia",
    "pr-id": "741",
    "commit-sha": "a10b81f123ee5a775a7a0f07fc347f344a18ebc7",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "backend",
    "pr-id": "1451",
    "commit-sha": "e694b4a5cae7940ad3cfc75fc876b18435fe697e",
    "pipeline": "master"
  }
]
```

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "aws-lb",
    "component": "backend",
    "pr-id": "1451",
    "commit-sha": "4d1ef1a7344f8b4dfa354f3b2333d29dc29d4f09",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "gaia",
    "pr-id": "741",
    "commit-sha": "a10b81f123ee5a775a7a0f07fc347f344a18ebc7",
    "pipeline": "master"
  }
]
```

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "aws-lb",
    "component": "gaia",
    "pr-id": "741",
    "commit-sha": "a10b81f123ee5a775a7a0f07fc347f344a18ebc7",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "backend",
    "pr-id": "1451",
    "commit-sha": "4d1ef1a7344f8b4dfa354f3b2333d29dc29d4f09",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "prismacloud",
    "pr-id": "69",
    "commit-sha": "4ad36337c1092f2dbfb862e1b15750e96e5e9e64",
    "pipeline": "master"
  }
]
```

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "aws-lb",
    "component": "gaia",
    "pr-id": "741",
    "commit-sha": "d374508e4307a11e68e92bb5a7055210a6f114f7",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "backend",
    "pr-id": "1451",
    "commit-sha": "532b9febeca96836ae2390743c5783c608d01e03",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "prismacloud",
    "pr-id": "69",
    "commit-sha": "4ad36337c1092f2dbfb862e1b15750e96e5e9e64",
    "pipeline": "master"
  }
]
```

**Review by dstiliadis — APPROVED** [review]

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "aws-lb",
    "component": "gaia",
    "pr-id": "741",
    "commit-sha": "d374508e4307a11e68e92bb5a7055210a6f114f7",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "backend",
    "pr-id": "1451",
    "commit-sha": "9b97597624737a13c8850f10a8ebc39ef5216c1c",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "prismacloud",
    "pr-id": "69",
    "commit-sha": "4ad36337c1092f2dbfb862e1b15750e96e5e9e64",
    "pipeline": "master"
  }
]
```

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "aws-lb",
    "component": "prismacloud",
    "pr-id": "69",
    "commit-sha": "4ad36337c1092f2dbfb862e1b15750e96e5e9e64",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "gaia",
    "pr-id": "741",
    "commit-sha": "d374508e4307a11e68e92bb5a7055210a6f114f7",
    "pipeline": "master"
  },
  {
    "project": "aws-lb",
    "component": "backend",
    "pr-id": "1451",
    "commit-sha": "2c59bf3e1ff5835d8ea003c87cd3ccf767681e38",
    "pipeline": "master"
  }
]
```

**Pull request merged**